doc/api/fs.md

```diff
@@ -3615,14 +3619,33 @@ It is unsafe to use `fs.writeFile()` multiple times on the same file without
 waiting for the callback. For this scenario, [`fs.createWriteStream()`][] is
 recommended.
 
-### File Descriptors
-1. Any specified file descriptor has to support writing.
-2. If a file descriptor is specified as the `file`, it will not be closed
-automatically.
-3. The writing will begin at the current position. For example, if the string
-`'Hello'` is written to the file descriptor, and if `', World'` is written with
-`fs.writeFile()` to the same file descriptor, the contents of the file would
-become `'Hello, World'`, instead of just `', World'`.
+### Using `fs.writeFile()` with File Descriptors
+
+When `file` is a file descriptor, the behaviour is almost identical to directly
+calling [`fs.write()`][] like:
+```javascript
+fs.write(fd, Buffer.from(options.encoding), callback);
+```
+
+The differences from directly calling `fs.write()` are:
+1. `fs.write()` only supports writing buffers, whereas `fs.writeFile()` supports
```

Comment:
What about [`fs.write(fd, string...)`](https://nodejs.org/api/fs.html#fs_fs_write_fd_string_position_encoding_callback) variant?

Reply:
good point, removed this, and no longer need a bulleted list.